app/views/shared/_error_messages.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if @errors[field]&.any? %>
+<% if @errors && @errors[field]&.any? %>
```

Comment:
Maybe `if @errors&.[](field)&.any?`?

Reply:
I don't prefer it but I've switched to it